cli/mfg.c

```diff
@@ -404,12 +412,149 @@ static int image_select(int argc, char **argv)
 	return ret;
 }
 
+static int fw_transfer(int argc, char **argv)
+{
+	const char *desc = "Transfer a firmware image to device (BL1 phase only)\n\n"
+			   "This command only supports transferring firmware "
+			   "image when device is in BL1 boot phase. To update "
```

Comment:
Please mention that this should be followed by a fw-execute command to activate the firmware and that the transferred firmware is stored in RAM and not preserved after a reboot.